app/controllers/application_controller.rb

```diff
@@ -1,8 +1,11 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   protect_from_forgery with: :exception
   helper_method :current_person
 
   def current_person
     @current_person ||= Person.find(session[:person_id]) if session[:person_id]
   end
+
+  alias pundit_user current_person
```

Comment:
Do we want to refer to this person as a `pundit_user`? Might be nicer to keep the gem's language out of our code.